website/snippets/snowflake-acct-name.md

```diff
@@ -0,0 +1,2 @@
+✅ Use: `db5261993,db5261993.east-us-2.azure` <br>
+❌ Avoid: `db5261993.eu-central-1.snowflakecomputing.com`
```

Comment:
It's giving this error during GitHub Actions continuous integration (CI):
```
6:44:36 AM: SyntaxError: /opt/build/repo/website/snippets/snowflake-acct-name.md: Expected corresponding JSX closing tag for <br>. (9:100)
```

I'm _guessing_ the fix _might_ be this:
```suggestion
✅ Use: `db5261993,db5261993.east-us-2.azure` <br/>
❌ Avoid: `db5261993.eu-central-1.snowflakecomputing.com`
```